src/settings.h

```diff
@@ -21,6 +21,7 @@
 
 #pragma once
 
+#include <QDir>
```

Comment:
move to .cpp, and init the lists in the ctor instead

generally - try to keep headers as minimal as possible to not blow up compile times unneccessarily